torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1074,6 +1274,22 @@ void ProcessTimeSeriesNode(Node* n) {
   }
 }
 
+void ProcessUnsqueezeNode(Node* n) {
+  if (TensorTypePtr output_type = n->output(0)->type()->cast<TensorType>()) {
```

Comment:
Let's reduce two levels of nested scopes:
```suggestion
  TensorTypePtr output_type = n->output(0)->type()->cast<TensorType>();
  if (output_type == nullptr) {
    return;
  }
```